admin/class-fastcgi-purger.php

```diff
@@ -90,8 +90,29 @@ public function purge_url( $url, $feed = true ) {
 				break;
 
 		}
+		
+		if( ( is_page() || is_single() ) && $nginx_helper_admin->options['purge_amp_urls'] ) {
+			$this->purge_amp_version( $url );
+		}
 
 	}
+	
+	/**
+	 * Purge AMP version of a URL.
+	 *
+	 * @param string $url_base The base URL to purge.
+	 */
+	private function purge_amp_version( $url_base ) {
+		$amp_url = sprintf( '%s/amp/', rtrim( $url_base, '/' ) );
+		
+		$this->log('- Purging AMP URL | ' . $amp_url);
```

Comment:
Fix the PHPCS error here.

```suggestion
		$this->log( '- Purging AMP URL | ' . $amp_url );
```